### ISSUE TITLE
Summarize the results from eval_agents

### ISSUE THREAD

**Issue by KeithEngland** [KeithEngland/RL-FinancialPlanning-Env#4]
Within EvalAgents.py, we create an output variable called "results." The purpose of this issue is to summarize those results.

For each agent_name, tell me:
  - the percentage of episodes that have a reward < 0 (ie, the percentage of episodes where the investor ended their life in financial ruin)
  - the average reward achieved by each agent_name
  - the average ending_age of each agent_name

Once that is complete, take all of the trained agents & group them by the settings used in their trainings. So if I had only one set of hyperparameters, but I trained 10 agents with different initializations, then group those 10 trained agents together. Agents with similar settings will have identical names except for the "counter." Then generate the same three data points above, grouping by training settings instead of agent_name, so that we can get average results by training settings. 

Bonus: create a simple visualization that shows the summary data for (1) each of the trained agents, (2) the average of the trained agents, and (3) the best 1-3 naive agents. This will give any readers a very clear understanding of how the trained agents do relative to the best naive agents. Ideally, this visualization will be saved on github somewhere so that it can be referenced easily by anybody interested in this project.
